ui/open-api-ui/pom.xml

```diff
@@ -39,6 +39,11 @@
             <artifactId>junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Is this really needed ?

Reply:
We change to assertThat its needed. assertThat show the error more friendly, its ok or should I remove?

Reply:
I am ok to introduce it. But maybe in another PR. Personally I would not add another dependency to manage, just for a nicer error message. If you want to have a custom error message you can pass the message in as the first argument (if I remember correctly) and I think you can also add `@DisplayName` on the method. @MikeEdgar w.d.y.t ? I do not like having multiple dependencies that does the same thing.

Reply:
Yeah, I tend to agree. I'm not apposed to assertj per se, but we're already using the plain Junit assertions throughout the project, other than the Gradle plugin (which is a different animal anyway). Better to be consistent on this.